source/query-with-natural-language/ai-and-data-usage-information.txt

```diff
@@ -0,0 +1,47 @@
+.. _compass-ai-data-usage:
+
+=============================
+AI and Data Usage Information
+=============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+
+Querying with natural language in Compass is powered by Generative AI
+(Gen AI), and may give inaccurate responses. See our `Generative AI FAQ 
+<https://dochub.mongodb.org/core/faq-ai-features>`__ 
+for more information about Gen AI in MongoDB products.
+
+Third Party Providers
+---------------------
+
+This feature currently uses the `Azure OpenAI Service
```

Comment:
Not a match to legal's edit (https://docs.google.com/document/d/1iHhXescsma0lkrSUQ49niehpJP1unehc4JkzgSuK5Xw/edit): 
```suggestion
Querying with natural language in Compass currently uses the `Azure OpenAI Service
```